dap_aria_mapping/pipeline/data_collection/processed_openalex.py

```diff
@@ -0,0 +1,361 @@
+import os
```

Comment:
this bit is needed for things to run in batch (it installs the specific requirements needed for the flow)